### PR TITLE
Add invoke task and docs about updating base image

### PR DESCRIPTION
**Pull request by hopson**
Documentation and some initial automation about update the base image in all repos.

### CODE REVIEW

File: tasks/__init__.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                edited_lines.append(f"ARG base_tag={base_tag}\n")
+            else:
+                edited_lines.append(line)
+    with open(filename, 'w') as file:
+        file.writelines(edited_lines)
+
+
+@task
+def update_repos(c, base_tag):
+    repos = parse_repos('./README.md')
```
Comment (billdirks) [inline tasks/__init__.py:47]: The README is code!

**Review by billdirks — APPROVED** [review]
LGTM!

It would be good to add in the PR conversation an example of a before/after of the changed lines in the Dockerfile. It would help me map the file editing done in code to the actual transformation.

**Review by TaylorBoudreau — APPROVED** [review]
LGTM! I'm not sure if this will get run anywhere aside from folks running it manually, but the repo list in the readme dependency seems potentially fragile, but probably not a big deal.

**Comment by hopson** [comment]
I am with you both about the README not being the best thing, but I also didn't want create another list without some more deliberate planning or dumping it in the submodule.

I am optimistic that just having a task for this is a step towards automating it.

**Pull request merged**